tests/scenarios/compat-app-html-attributes-test.ts

```diff
@@ -59,9 +59,9 @@ appScenarios
         expectFile('./index.html').doesNotMatch('rel="stylesheet"', 'does not have rel=stylesheet');
         expectFile('./index.html').matches('<script defer', 'has script defer');
         expectFile('./index.html').doesNotMatch('<script src', 'does not have script src');
-        expectFile('./index.html').doesNotMatch(
+        expectFile('./index.html').matches(
           'data-original-filename="vendor.css">',
-          'does not have data-original-filename vendor.css'
+          'has data-original-filename vendor.css'
```

Comment:
The compat-app-builder used to compute a boolean `implicitStyles` and add the stylesheet only if `true`. With this PR, we simplify this and add the stylesheet without condition.